scripts/package_build.py

```diff
@@ -66,6 +66,17 @@ def get_relative_path(source_dir, target_file):
         target_file = target_file.replace(source_dir, "").lstrip('/')
     return target_file
 
+def git_commit_hash():
+    return subprocess.check_output(['git','log','-1','--format=%h']).strip().decode('utf8')
+
+def git_dev_version():
+    version = subprocess.check_output(['git','describe','--tags','--abbrev=0']).strip().decode('utf8')
+    long_version = subprocess.check_output(['git','describe','--tags','--long']).strip().decode('utf8')
+    version_splits = version.lstrip('v').split('.')
+    dev_version = long_version.split('-')[1]
+    version_splits[2] = str(int(version_splits[2]) + 1)
+    return '.'.join(version_splits) + "-dev" + dev_version
```

Comment:
I don't know if it's the right time to address this right now or right here. However, when running this "on the tag", like
```
git checkout v0.2.2
git describe --tags --long | cut -f2 -d-
0
```

This routine might want to account for that and not add the `"-dev" + dev_version` suffix?

Reply:
Yeah that makes sense. I will add that in.

Reply:
So, will the magic for bumping to the `NEXT_VERSION` brought up by @hannesmuehleisen at https://github.com/cwida/duckdb/pull/1176#issuecomment-737300866 right here?

If so, the bumping through `str(int(version_splits[2]) + 1)` might also not happen when `git describe --tags --long | cut -f2 -d-` yields zero.

By doing so, we might be able to get around doing it manually like outlined at https://github.com/cwida/duckdb/pull/1176#issuecomment-737310975:
- If this is invoked on any gitref not aligned with a GA tag, it will yield a bumped version on the patchlevel designating the next development iteration.
- If this is invoked on a gitref aligned with a GA tag, it will use this very tag without any manipulation.

Reply:
Have just implemented exactly this ^^